mungegithub/submit-queue/deployment/docs/configmap.yaml

```diff
@@ -7,7 +7,7 @@ data:
   config.http-cache-dir: /cache/httpcache
   config.organization: kubernetes
   config.project: kubernetes.github.io
-  config.pr-mungers: size
```

Comment:
We cannot turn it on before the check-labels munger creates the size labels for this repository. We'll turn it back on after that is done.